qface/idl/domain.py

```diff
@@ -335,7 +335,8 @@ def minorVersion(self):
     @property
     def module_name(self):
         """ returns a capitalized name from the module name """
-        return self.name.split('.')[-1].capitalize()
+        str = module.name.split('.')[-1]
```

Comment:
looking at this code, I would think we should not capitalise anything and just return the last part of the module uri. The capitilisation should be done during generation.